install/kubernetes/cilium/values.yaml

```diff
@@ -755,6 +755,8 @@ cni:
       memory: 10Mi
   # -- Enable route MTU for pod netns when CNI chaining is used
   enableRouteMTUForCNIChaining: false
+  # -- Enable the removal of iptables rules created by the AWS CNI VPC plugin.
+  iptablesRemoveAWSRules: true
```

Comment:
Please apply this change to `values.yaml.tmpl` and then re-generate the docs by running

```
make -C install/kubernetes && make -C Documentation update-helm-values
```

Reply:
Thanks. Done.